frontend/src/properties/propertysections/ChartWidgetPropertySection.tsx

```diff
@@ -53,197 +47,3 @@ const isBarChart = (chart: Chart): chart is BarChart => {
 const isPieChart = (chart: Chart): chart is PieChart => {
   return chart.metadata?.kind === 'PieChart';
```

Comment:
`metadata` cannot be `null` here. Do the change four lines above too.

Reply:
done